lxd/db/certificates.go

```diff
@@ -134,13 +135,44 @@ type CertificateFilter struct {
 func (c *Cluster) GetCertificate(fingerprint string) (*Certificate, error) {
 	var err error
 	var cert *Certificate
+	objects := []Certificate{}
 	err = c.Transaction(func(tx *ClusterTx) error {
-		cert, err = tx.GetCertificate(fingerprint + "%")
+		sql := `
+SELECT certificates.id, certificates.fingerprint, certificates.type, certificates.name, certificates.certificate, certificates.restricted
+FROM certificates
+WHERE certificates.fingerprint LIKE ?
```

Comment:
As the certificate is being matched on fingerprint with a LIKE shouldn't the argument be called fingerprintPrefix like the images function?